templates/typescript/webcrypto.ts

```diff
@@ -0,0 +1,66 @@
+let subtle = self.crypto.subtle;
```

Comment:
In most of the code below, you call `self.crypto.subtle` directly rather than using this shortcut. That seems fine, and it doesn't make the code hard to follow! I'd suggest getting rid of this variable declaration, and changing its usages below (on lines 29 and 47) to reference `self.crypto.subtle` directly. In this kind of example content, I think being clear and using the underlying API as directly as possible is best.

Reply:
Done.